### ISSUE TITLE
crd-install warning with glooe v1.6.x helm install

### ISSUE THREAD

**Issue by kevin-shelaga** [solo-io/gloo#4694]
**Describe the bug**
When installing glooe v1.6.x with helm 3 users receive `crd-install` helm hook warnings

**To Reproduce**

1.
```sh
helm install gloo glooe/gloo-ee --namespace gloo-system \
  --create-namespace --set-string license_key=$KEY --version v1.6.10
```

2.
```sh
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
```

**Expected behavior**
No warnings

**Additional context**
Add any other context about the problem here, e.g.
- Gloo Edge version - v1.6.x
- Kubernetes version - 1.19.8

**Comment by sam-heilbron** [comment]
Related comment: https://github.com/prometheus-community/helm-charts/issues/155#issuecomment-706743032

To confirm, this produces warnings but no errors? Does this also occur in Gloo OS? 
This is related to the fact that in 1.6.x we support helm2 and helm3. This warning exists because helm3 deprecated the `crd-install` hook. Here's the helm issue to track this: https://github.com/helm/charts/issues/19008. I believe we can close this issue if it just provides warnings. Does that work with you?

**Comment by kevin-shelaga** [comment]
Will keep for tracking but this is fixed in 1.7.x+
